lib/mongoose_push/application.ex

```diff
@@ -206,4 +208,36 @@ defmodule MongoosePush.Application do
         )
     end
   end
+
+  defp check_apns_ciphers() do
+    all_ciphers = :ssl.cipher_suites()
+
+    apns_ciphers =
+      Enum.filter(all_ciphers, fn cipher ->
+        case cipher do
+          {:ecdhe_rsa, :aes_128_gcm, _, :sha256} ->
+            true
+
+          {:ecdhe_rsa, :aes_256_gcm, _, :sha384} ->
+            true
+
+          _ ->
+            false
+        end
+      end)
```

Comment:
Just a cosmetic thing. It looks like to me, that `Enum.any?` could be used instead of `Enum.fiter`, thanks to that we'll have the answer immediately, without the need to calculate the length of the new list.

Reply:
Corrected. But in exchange we've lost the underscore and have to match to `:aead` directly.

Reply:
I think the previous `fun` used with `filter` would work as well. Anyway, thanks for applying my suggestion.